instill/clients/model.py

```diff
@@ -315,6 +527,9 @@ def list_models(
                         page_token=next_page_token,
                         show_deleted=show_deleted,
                         view=model_definition_interface.VIEW_FULL,
+                        # filter=filter,
+                        # visibility=visibility,
```

Comment:
we still has [these fields](https://github.com/instill-ai/protobufs/blob/main/model/model/v1alpha/model.proto#L339-L344)

Reply:
should we give them a default value?

Reply:
we can leave filter empty, since its not required. and default `visibility` to Public

Reply:
updated